app/models/seller.rb

```diff
@@ -5,6 +5,7 @@ class Seller < ApplicationRecord
 
   validates_inclusion_of :founded_year, :in => 1800..2020
   validates_uniqueness_of :seller_id
+  validates_uniqueness_of :square_location_id
```

Comment:
also wanna validate the presence of too :) 

https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_model/validate_presence_of_matcher.rb

Reply:
Haha no worries